future/transaction.go

```diff
@@ -597,20 +597,22 @@ func byte64FromBase64(in string) (out [64]byte, err error) {
 //                 maintained inside the account of any users who opt into this
 //                 application. The LocalStateSchema is immutable.
 //
+// - extraPages    ExtraProgramPages specifies the additional app program size requested in pages.
+//                 A page is 1024 bytes. This field enables execution of app programs
+//                 larger than the default maximum program size.
+//
 // - onComplete    This is the faux application type used to distinguish different
 //                 application actions. Specifically, OnCompletion specifies what
 //                 side effects this transaction will have if it successfully makes
 //                 it into a block.
 //
-// - extraPages    ExtraProgramPages specifies the additional app program size requested in pages.
-//                 A page is 1024 bytes. This field enables execution of app programs
-//                 larger than the default maximum program size.
-//
 // - boxes         lists the boxes to be accessed during evaluation of the application
 //                 call. This also must include the boxes accessed by inner app calls.
 
 // MakeApplicationCreateTx makes a transaction for creating an application (see above for args desc.)
 // - optIn: true for opting in on complete, false for no-op.
+//
+// NOTE: if you need to use extra pages or boxes, use MakeApplicationUpdateTxWithBoxes instead.
```

Comment:
I think @michaeldiamant 's comment also apply here tho...

Reply:
Thanks, fixed in 9b8ad96a29b8b01fe3ddf129e0249f4f2b7f6e18